sample_calendar.json

```diff
@@ -1,25 +1,38 @@
 {
     "calendars" : {
-        "checkit_148460826559529600" : {
+        "checkit_148460826559529600" { :
+            {
             "startDateString":"20170101",
             "endDateString":"20170202",
             "title":"hi",
             "uniqueId":148460826559529600,
             "checkedDays":{}
             },
+            "permissionRead": "JULxYGaHuFYlSkXqTC2Fx9KIofp2",
+            "permissionWrite": "JULxYGaHuFYlSkXqTC2Fx9KIofp2"
```

Comment:
I could also do:
"permissionRead" : {
    "JULxYGaHuFYlSkXqTC2Fx9KIofp2" : "true"
}

Although, I'm not sure if that would make sense? I mean, will it ever be false?